src/components/NcSettingsSection/NcSettingsSection.vue

```diff
@@ -154,7 +154,7 @@ $maxWidth: 900px;
 		// make sure to properly align the icon with the text
 		margin: -$icon-margin;
 		margin-left: 0;
-		opacity: $opacity_normal;
+		color: var(--color-text-lighter);
```

Comment:
This variable is deprecated and will shortly removed, use its replacement instead:
```suggestion
		color: var(--color-text-maxcontrast);
```